### DISCUSSION TITLE
Adding a company

### DISCUSSION THREAD

**Discussion by zer0fuxs** [remoteintech/remote-jobs#1623] [General]
Hey everyone :)
I'm pretty new to github and dont have much experience with coding. I dont know how to add a company but would like to, can someone help me please? 

Thaaaaanks in advance

**Comment by douglasdcm** [comment]
I'm new here too, so I never added companies here. I'd do the same thing was done in this MR. A new company was added to README.md and a new <company>.md file was added to company-profiles folders
https://github.com/remoteintech/remote-jobs/commit/993a8f92b1a098f6bf06b5094a5815862341008a

**Comment by Ki-er** [comment]
Hi there,

Apologies for the *very* long delay. 

No coding experience is needed for this repo! You just need to fork the code, create a .md file with the title using the [example](https://github.com/remoteintech/remote-jobs/blob/main/company-profiles/example.md) file as a base. All of this can be done using the web browser. Check out the [contributing]()https://github.com/remoteintech/remote-jobs/blob/main/.github/CONTRIBUTING.md file for further guidance.

Here is a couple of guides: 

- [How to fork](https://docs.github.com/en/get-started/quickstart/fork-a-repo)
- [Create file](https://docs.github.com/en/repositories/working-with-files/managing-files/creating-new-files)
- [Create a PR](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request)

**Comment by picklow526152-wq** [comment]
The guidance on forking a repository, creating a company profile, and submitting a pull request makes the process much more approachable for newcomers. Community-maintained directories stay valuable when more people contribute updates and new listings. I’ve noticed a similar collaborative approach on job and [company listing ](https://www.pakistanjobs.pk/company/)platforms .
